onnxruntime/contrib_ops/cuda/quantization/qordered_ops/qordered_attention.cc

```diff
@@ -185,7 +185,7 @@ QOrderedAttention::QOrderedAttention(const OpKernelInfo& info) : CudaKernel(info
 
 #else
 
-  ORT_ENFORCE(false, "Compiling with CUDA_VERSION >= 11.4 is needed!")
+  ORT_ENFORCE(false, "Compiling with CUDA_VERSION >= 11.4 is needed!");
```

Comment:
It should be:

···
#error "Compiling with CUDA_VERSION >= 11.4 is needed!"
···

A compile time error is better?

Reply:
Compile time error means if anyone build with CUDA 113 it will not be compiled. So we are not allowing anyone to use CUDA 113 from now? even they checkout the code and build by themselves?

Reply:
I think the better way is the code owner can think of putting all related code/class/files for cuda>=114 only, instead of using the macro inside the functions.

Reply:
yes, we should continue to support CUDA 11.3 for users building from source. I think there may be older hardware that is only supported on older CUDA version. 
Since CUDA 11.4 is only required for a few of these contrib kernels, it seems like this should be supported/organized in a better way.